moped-database/migrations/1697679713991_confirmed_date_trigger/up.sql

```diff
@@ -0,0 +1,25 @@
+-- For each of phase_start and phase_end date, if the date is on or before 
+-- the current date when the record is created or edited then set corresponding
+-- confirmation column to true.
+CREATE OR REPLACE FUNCTION moped_proj_phases_confirmed_dates() RETURNS trigger
+    LANGUAGE plpgsql
+    AS $$ 
+    BEGIN
+        IF NEW.phase_start <= current_date AND NEW.is_phase_start_confirmed != TRUE THEN
+            new.is_phase_start_confirmed := true;
+        END IF;
+        IF NEW.phase_end <= current_date AND NEW.is_phase_end_confirmed != TRUE  THEN
+            new.is_phase_end_confirmed := true;
+        END IF;
+        IF NEW.phase_start > current_date THEN
```

Comment:
here, too, you could add the second condition, e.g. `and NEW.is_phase_start_confirmed != FALSE`